src/c_types.py

```diff
@@ -78,10 +78,16 @@ class Function:
     is_variadic: bool
 
 
+@dataclass(eq=False)
+class Enum:
+    tag: Optional[str]
+    values: Dict[int, str]
```

Comment:
"values" sounds like a mapping of str to int to me, would prefer "names"/"labels"/similar (also in naming like "get_enum_value")